bread/layout/components/search_select.py

```diff
@@ -0,0 +1,109 @@
+import htmlgenerator as hg
+from django.utils.translation import gettext_lazy as _
+
+import bread.utils
+
+from .icon import Icon
+from .loading import Loading
+from .tag import Tag
+
+
+class SearchSelect(hg.BaseElement):
+    def __init__(
+        self,
+        search_view,
+        query_urlparameter="q",
+        size="xl",
+        searchinput_widgetattributes=None,
+        widgetattributes=None,
+        **elementattributes,
+    ):
+        current_selection_id = widgetattributes["value"][0]
+        # This works inside a formset. Might need to be changed for other usages.
+        current_selection = (
+            getattr(
+                elementattributes["boundfield"].form.instance,
+                elementattributes["fieldname"],
+            )
+            if current_selection_id
+            else None
+        )
+        del elementattributes["boundfield"]
+        elementattributes["_class"] = (
+            elementattributes.get("_class", "") + f" bx--search bx--search--{size}"
```

Comment:
The class ```bx--search--{size}``` should be on the input element, not on the outer element. I saw this was originally my fault, done in ```search.py```.

Reply:
are you sure? see screenshot from the official examples
![grafik](https://user-images.githubusercontent.com/11404846/122058564-07b1c980-cdec-11eb-9b1c-2eab02c7e42a.png)

Reply:
Sorry, my confusion. I misread the SCSS definition.